app/controllers/appoinment_bookings_controller.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+# users service  booking controller
+class AppoinmentBookingsController < ApplicationController
+  def index
+    @opts = if params[:vehicle] == 'two'
+              AppoinmentBooking::Two_wheeler
+            else
+              AppoinmentBooking::Four_wheeler
+            end
+            @bookedappoinment = AppoinmentBooking.where(user_id: params[:id])
+  end
+
+  def new
+    @bookappoinment = AppoinmentBooking.new
+  end
+
+  def create
+    @bookappoinment = AppoinmentBooking.new(allowed_parameter)
+    @bookappoinment.user_id = current_user.id
```

Comment:
current_user.appointment_bookings.build
```suggestion
    current_user.appointment_bookings.build
```